src/dashboard/src/main/models.py

```diff
@@ -838,6 +838,26 @@ class Meta:
         # set up permissions: https://code.djangoproject.com/ticket/18866
         verbose_name = u'ASDIPObjectResourcePairing'
 
+class ArchivesSpaceDOComponent(models.Model):
+    """
+    Represents a digital object component to be created in ArchivesSpace at the time an AIP is stored by Archivematica.
+
+    In ArchivesSpace, a digital object component is meant to be parented to a digital object record.
+    The workflow in use by the appraisal tab doesn't expose digital objects to the user, just components;
+    one digital object should be created as a parent for these components before creating the
+    components themselves.
+    """
+    id = models.AutoField(primary_key=True, db_column='pk')
```

Comment:
Django will create the ID field automatically with db_column='id' https://docs.djangoproject.com/en/1.7/topics/db/models/#automatic-primary-key-fields  Is there a reason to call the column 'pk' instead of letting Django's defaults handle it?